src/utils/HypervBoxUtil.js

```diff
@@ -54,9 +51,8 @@ var HypervBox = {
     // there seems to be a problem with elevated execution. see: https://github.com/nodejs/node-v0.x-archive/issues/6797
     // The only easy possibility seems to be to communicat with a file.
 
-      return util.execFile([this.command(), 'Get-VMHostSupportedVersion']).then(stdout => {
-
-        let match = stdout.match(/^(.*) True/im);
+      return util.execFile([this.command(), 'Get-VM | Where {$_.Name -eq "default" -and $_.Version} | select Version']).then(stdout => {
```

Comment:
This might be done as a future pull request as default if hard coded all over the place now.